src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs

```diff
@@ -771,7 +782,9 @@ private Task<GetAccountTransferConnectionsResponse> GetTransferConnectionsNoAuth
                         WithProviderCount = commitmentStatuses.Count(m =>
                              m == RequestStatus.WithProviderForApproval
                           || m == RequestStatus.SentToProvider
-                          || m == RequestStatus.SentForReview)
+                          || m == RequestStatus.SentForReview),
+                        TransferFundedCohortsCount = commitmentStatuses.Count(m =>
+                            m == RequestStatus.WithSender)
```

Comment:
Ho do we know it's with the sender? What sets this RequestStatus?

Reply:
Again, this code has already been refactored in DPP-1229 - https://github.com/SkillsFundingAgency/das-employercommitments/pull/117. In the refactored code, the RequestStatus is calculated in the GetStatus() call and WithSender has been split into 2 separate statuses.